### DISCUSSION TITLE
Option to pre-populate a form when generating a link

### DISCUSSION THREAD

**Discussion by SebastianJurgenson** [team-forms/community#5] [Ideas]
Hi All,

It would be very useful if we can generate a unique link to a form which pre-populates the form with different data based on the link. For example I could create a unique link for each person to pre-populate there email.

**Comment by Erin-Dwyer** [comment]
Hi @SebastianJurgenson,

The avility to generate a pre-filled link to a form was recently released :) Please see the article below on how this feature works.

[Create pre-filled share links](https://help.teamforms.app/en/articles/8885675-create-pre-filled-share-links
)
